core/commands/add.go

```diff
@@ -71,6 +71,31 @@ its filename. For example:
 You can now refer to the added file in a gateway, like so:
 
   /ipfs/QmaG4FuMqEBnQNn3C8XJ5bpW8kLs7zq2ZXgHptJHbKDDVx/example.jpg
+
```

Comment:
different wording
The chunker option, '-s', specifies the chunking strategy and size for how to store and deduplicate larger files into blocks.

Reply:
maybe just this? 
'...specifies the chunking strategy that dictates how to break files into blocks'

Reply:
Yes that's better, should be updated in last commit